server.js

```diff
@@ -31,42 +38,44 @@ app.get('/books', async (request, response) => {
   response.send(books);
 });
 
-app.listen(PORT, () => console.log(`listening on ${PORT}`));
-
-
-
-
-// app.get('/test', (request, response) => {
-//   response.send('test request received');
-// });
-
-
-// const BookModel = mongoose.model('book-collection', bookSchema);
-
-// const book1 = new BookModel(
-//   'Being and Nothingness',
-//   'An Essay on Phenomenological Ontology',
-//   'read',
-//   'bookreader1@gmail.com'
-// );
+app.post('/books', async (request, response) => {
+  const newBook = await Book.create({
+    ...request.body,
+    email: request.query.email,
+  });
+  response.send(newBook);
+});
```

Comment:
Feedback here: Consider adding in some error handling here as well

- send status of 200 if success or if something goes wrong consider sending different status 👍